src/org/daisy/dotify/formatter/impl/core/TableOfContentsImpl.java

```diff
@@ -234,9 +235,14 @@ public void newLine() {
     }
 
     @Override
-    public void insertReference(String identifier, NumeralStyle numeralStyle) {
+    public void insertPageReference(String identifier, NumeralStyle numeralStyle) {
         assertInEntry();
-        super.insertReference(identifier, numeralStyle);
+        super.insertPageReference(identifier, numeralStyle);
+    }
+
+    @Override
+    public void insertMarkerReference(MarkerReference ref, TextProperties t) {
```

Comment:
Can you please add `/* .... */` with an explanation why this is unsupported?

Reply:
Fixed.